src/styles/styles.js

```diff
@@ -32,6 +33,8 @@ import Colors from './colors';
 // touchCallout is an iOS safari only property that controls the display of the callout information when you touch and hold a target
 const touchCalloutNone = Browser.isMobileSafari() ? {WebkitTouchCallout: 'none'} : {};
 
+const lineHeightBadge = getPlatform() === CONST.PLATFORM.WEB ? {lineHeight: variables.lineHeightXSmall} : {lineHeight: variables.lineHeightNormal};
```

Comment:
Previously It was safari-specific (It causes issues on Android) as far as I remember. Why we are changing it to be web-only then? chrome has no problem with centering text 🤔